iaso/api/completeness_stats.py

```diff
@@ -271,6 +282,11 @@ def list(self, request: Request, *args, **kwargs) -> Response:
         if users:
             instance_qs = instance_qs.filter(created_by__in=users)
 
+        # filter instance_qs and form_qs on projects
+        if projects:
+            instance_qs = instance_qs.filter(project__in=projects)
+            form_qs = form_qs.filter(projects__in=projects).distinct()
```

Comment:
Same here.

If possible, I would use something like `distinct("id")` to avoid a potential performance issue.